kmod/muser.c

```diff
@@ -1119,10 +1119,10 @@ static unsigned int get_minsz(unsigned int cmd)
 	case VFIO_DEVICE_SET_IRQS:
 		return offsetofend(struct vfio_irq_set, count);
 	}
-	return -1;
+	return -EOPNOTSUPP;
```

Comment:
`EINVAL` would be more suitable.

Reply:
The invalid(EINVAL) cmds are already checked at vfio layer, vfio only sends us the valid commands. It is at our layer either we support them or not, that is the reason EOPNOTSUPP looks more better.

Reply:
OK, I think it's worth a comment in our code that VFIO does this.